.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1 +1,14 @@
 blank_issues_enabled: false
+contact_links:
+  - name: Auth0 Community
+    url: https://community.auth0.com/tags/c/sdks/5
```

Comment:
Is this supposed to be a public available URL?

Reply:
Corrected